lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -447,6 +450,8 @@ def test_get_group_query(self):
         self.assertSetEqual(correct_students, check_students)
         students_left = StudentsForQuery.objects.all()
         queries_left = TemporaryQuery.objects.all()
+
+        delete_group_temp_queries_and_students(query_group)
```

Comment:
I think it flows more logically if this is called before `students_left` and `queries_left` are assigned, even though functionally both work